styles/_theme.scss

```diff
@@ -276,6 +280,12 @@ a.cg-arrow {
   }
 }
 
+.bg-primary-darker {
+  h1 {
```

Comment:
USWDS offers utilities for setting font-weight: https://designsystem.digital.gov/utilities/text-styles/#font-weight-2

If it really must be applied everywhere, universally, then include the font weight mixin. If it's just for one place in one template, add it as a utility class.